src/System.Private.Xml/tests/Xslt/XslTransformApi/DataHelper.cs

```diff
@@ -110,20 +110,20 @@ public String PublicFunction()
         return "Inside Public Function";
     }
 
-    private String PrivateFunction()
-    {
-        return "Inside Private Function";
-    }
+//    private String PrivateFunction()
+//    {
+//        return "Inside Private Function";
+//    }
 
-    protected String ProtectedFunction()
-    {
-        return "Inside Protected Function";
-    }
+//    protected String ProtectedFunction()
+//    {
+//        return "Inside Protected Function";
+//    }
 
-    private String DefaultFunction()
-    {
-        return "Default Function";
-    }
+//    private String DefaultFunction()
+//    {
+//        return "Default Function";
+//    }
```

Comment:
Typically functions like this are used via reflection.  Do we know they're not being used?